app/components/views/TicketsPage/GovernanceTab/Page.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import AgendaCard from "AgendaCard";
+import AgendaOverview from "AgendaOverview";
+import SelectStakePool from "SelectStakePool";
+import { FormattedMessage as T } from "react-intl";
+import "style/StakePool.less";
+
+const VotingPrefsPage = ({
+  agendas,
+  stakePool,
+  selectedAgenda,
+  configuredStakePools,
+  onChangeStakePool,
+  getAgendaSelectedChoice,
+  onShowAgenda,
+  onCloseAgenda,
+  onUpdateVotePreference
+}) => (
+  <div className="tab-card">
+    <div className="stakepool-voting-title-area">
+      <div className="stakepool-voting-title-area-name">
+        <T id="votingPreferences.title" m="Voting Preferences" />
+      </div>
+      {configuredStakePools.length > 0 ?
+      <div className="stakepool-unconfigured-select">
+        <SelectStakePool
+          options={configuredStakePools}
+          value={stakePool}
+          onChange={onChangeStakePool}
+        />
+      </div> :
+      <div></div>
+      }
+    </div>
+    <div className="stakepool-voting-agenda-area">
+      {selectedAgenda ? (
```

Comment:
this is really minor, but we don't need to use the { query ? <Component/> : null } pattern anymore.

react 16 allows 'false' as a valued return for a component now, so you can just do { query && <Component/> }